modules/Dasboard/components/ClassroomCardList/ClassroomCardList.tsx

```diff
@@ -0,0 +1,26 @@
+import { Grid } from "@mantine/core";
+
+import { useGetClassroomsQuery } from "@/shared/redux/rtk-apis/classrooms/classrooms.api";
+
+import ClassroomCard from "../ClassroomCard/ClassroomCard";
+import { Classroom } from "./ClassroomCardList.types";
+
+const ClassroomList = () => {
+  const { data: classrooms, isLoading, error } = useGetClassroomsQuery();
+
+  if (isLoading) return <p>Loading classrooms...</p>;
+  if (error) return <p>Error loading classrooms</p>;
+  if (!classrooms || classrooms.length === 0) return null;
```

Comment:
Not sure if I understand what's happening here. If there are no classrooms, would the users see an empty screen?

Reply:
No bhaiya, the user will see the CreateClassroomButton, but my implementation was wrong here, I have updated my code.
Initially I was fetching data from this component which is not needed, I could just fetch the classroom data from the dashboard and pass the data in this component.